pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -1871,6 +1872,20 @@ def _show_mountpoint(self, page=None, mountpoint=None):
                 self.on_selector_clicked(member)
                 break
 
+    def _get_pages_with_device(self, dev):
+        pages = []
+
+        for page in self._accordion.allPages:
+            for member in page.members:
+                if dev == member._device:
+                    pages.append(page)
+                    break
+
+        return pages
```

Comment:
This could be optimized in a way that you will look for just two pages and kill the cycle. There could be systems where this cycle may take a big amount of time.